### PR TITLE
fix pangolin tests

### PR DESCRIPTION
**Pull request by bernt-matthias**
output seems to have changed

not sure if the new assertions will be useful in the future. opinions?

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/pangolin/pangolin.xml [hunk 1]
```diff
@@ -103,7 +103,7 @@
             </conditional>
             <output name="output1" ftype="tabular">
                 <assert_contents>
-                    <has_text_matching expression="B.1.1\t0.0\t*PUSHER" />
+                    <has_text_matching expression="B\.1\.1.*\t0\.0\t*PUSHER" />
```
Comment (bgruening) [inline tools/pangolin/pangolin.xml:106]: How could the test pass initially? I don't understand those failures :(

**Review by bgruening — APPROVED** [review]
Thanks for fixing it!

File: tools/pangolin/pangolin.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/pangolin/pangolin.xml:106]: I think pangolin uses an online database in the tests. This seems to change over time.

Comment (bgruening) [reply tools/pangolin/pangolin.xml:106]: ah, the missing backslashes are not the problem but the missing `.`. Ok got it.

Comment (bernt-matthias) [reply tools/pangolin/pangolin.xml:106]: Maybe we should add `.*` it in all tests?

Comment (bgruening) [reply tools/pangolin/pangolin.xml:106]: @pvanheus do you have an idea.

It would make the test less stringent I think. But if its needed let's do it.

**Pull request merged**